xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-ui/src/main/resources/Refactoring/Code/RefactoringTranslations.xml

```diff
@@ -38,8 +38,8 @@
   <hidden>true</hidden>
   <content>refactoring.configuration.delete.heading=Delete Properties
 admin.delete=Delete
-Refactoring.Code.RefactoringConfigurationClass_canSkipRecycleBin=Can skip the recycle bin
-Refactoring.Code.RefactoringConfigurationClass_canSkipRecycleBin.hint=When set to Yes, allows advanced users to choose whether they want to send pages to the recycle bin or delete them permanently when deleting a page. When set to No, pages are always sent to the recycle bin if it is available, otherwise they are always permanently deleted. The default value is No.</content>
+Refactoring.Code.RefactoringConfigurationClass_isRecycleBinSkippingActivated=Can skip the recycle bin
+Refactoring.Code.RefactoringConfigurationClass_isRecycleBinSkippingActivated.hint=When set to Yes, allows advanced users to choose whether they want to send pages to the recycle bin or delete them permanently when deleting a page. When set to No, pages are always sent to the recycle bin if it is available, otherwise they are always permanently deleted. The default value is No.</content>
```

Comment:
` when deleting a page` --> `when deleting pages` (there can be more than one and I expect all of them to be sent to the recycle bin for ex).

`if it is available` is not very clear. I'd say something like `if it is enabled` or `if it is active`

Actually this makes me think that we know if the recycle bin is enabled or not so why do this? Why not instead only display this config option if the recycle bin is active instead?

Reply:
Updated. The hint will be more clearer and simple once https://jira.xwiki.org/browse/XWIKI-17821 will be done.